apps/antalmanac/src/stores/AppStore.ts

```diff
@@ -6,6 +6,19 @@ import { Schedules } from './Schedules';
 import { SnackbarPosition } from '$components/NotificationSnackbar';
 import { CalendarEvent, CourseEvent } from '$components/Calendar/CourseCalendarEvent';
 import { useTabStore } from '$stores/TabStore';
+import actionTypesStore from '$actions/ActionTypesStore';
+import {
```

Comment:
This isn't enforced by the tsconfig, but prefer to indicate when things are being imported specifically for their type information, vs. as actual JS entities.

```suggestion
import type {
```